dts/arm/nordic/nrf54l_common.dtsi

```diff
@@ -18,7 +18,7 @@
 			rram0: rram@0 {
 				compatible = "soc-nv-flash";
 				erase-block-size = <4096>;
-				write-block-size = <1>;
+				write-block-size = <16>;
```

Comment:
Replace this PR description, fromlist -> fromtree, add the commit it is from and remove the upsteeam PR link